inversegame.js

```diff
@@ -1,61 +0,0 @@
-//this has a several bugs (some with the 'if'statements/loops and maybe some type coersion errors) but technically works
-//would've liked to have had more time with it though
-
-const { type } = require('os');
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-
-
-
-start();
-
-// declaration of functions + variables im going to use throughout
-function randNum(min, max) {
-    let range = (max - min) + 1;
-    let decimal = (Math.random() * range);
-    let int = Math.floor(decimal) + min;
-    return int;
-}
-
-
-
-async function start() {
-    console.log("Let's play a game where I (computer) make up a number between 1 and 100 and you (human) try to guess it.")
-
-    const secretNumber = randNum(1, 100) 
-
-    await ask("My secret number is: " + secretNumber + " Hit enter to proceed")
```

Comment:
This line doesnt need to be here, the computer guess will be stored in the variable established on line 31, if you wanted to have a `console.log(secretNumber)`, this would tell you what the number is so that you can make sure it works but not require the user to do anything to keep the game moving forward.